examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb

```diff
@@ -57,7 +57,7 @@
     "HugeCTR is a GPU-accelerated recommender framework designed to distribute training across multiple GPUs and nodes and estimate Click-Through Rates (CTRs).<br>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %%writefile train_hugeCTR.py
</pre><p>Looks like the file name is not <code>model.py</code> here. it is <code>train_HugeCTR.py</code>  so better to modify the file name in the sentence above:   <code>We will write the model to ./model.py and execute it afterwards</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
corrected